homework-2/scss/_sliderlogo.scss

```diff
@@ -0,0 +1,82 @@
+@import 'vars';
+@import 'mixins';
+
+.slider-logo {
+  padding: 100px 0;
+  .container{
+    @include widescreen {
+      padding: 0 200px;
+    }
+  }
+  @include widescreen {
+    padding: 145px 0;
+  }
+}
+.slider-logo-single {
+  .slick-slide .single-img {
+    max-width: 235px;
+    width: 100%;
+    margin: 0 auto;
+  }
+  .slick-disabled {
+    opacity: 0.5;
+  }
+}
+.slider-logo-list {
+  margin-top: 40px;
+  .img-logo { 
+    max-width: 70px;   
+    width: 100%;
+    margin: auto;
+    padding-bottom: 15px;
```

Comment:
Images not supposed to have paddings. You have wrapper - add padding there. But no need to add same padding to all elements in slider - add one padding to common wrapper.

Reply:
add padding img wrapper, because common wrapper broken my slider underline